projects/table-builder/src/lib/directives/styler.ts

```diff
@@ -4,15 +4,13 @@ import { Directive,  Input, ElementRef } from '@angular/core';
 @Directive({
     selector: '[styler]',
 }) export class StylerDirective {
-  @Input() styler: any;
+  @Input() set styler(styles){

```

Comment:
Now that I'm thinking about it.
We can put this back in ng on init.

Then we can update the styles on the element in the directive directly without the styler directive.
Then on next page the new styles will get applied since it was sent to the store.

Reply:
I would rather this way. I removed the direct styling. I don't think it amkes a difference. I would rather keep it one way as much as possible